includes/ApplHandlingByID.php

```diff
@@ -434,11 +436,13 @@ function forwardAcceptor ( $id, $type, $user, $next )
 				die();
 			}
 			
+			$next = str_replace('.','$',$next);
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* No space found after comma in function call

<!-- Comment by [SideCI](https://sideci.com) -->